docs/tutorials/configure-external-signer-tls.md

```diff
@@ -22,9 +22,9 @@ This tutorial configures TLS between Teku and Web3Signer, and use the [`keytool`
 - [Web3Signer signing key files](https://docs.web3signer.consensys.net/en/latest/HowTo/Use-Signing-Keys/) for validators on the testnet.
 - [Teku Installed](../get-started/install/install-binaries.md).
 - [Java `keytool`](https://docs.oracle.com/en/java/javase/12/tools/keytool.html).
-- A running execution client such as [Hyperledger Besu], or cloud-based service such as [Infura] synced to the Goerli testnet.
+- A running execution client such as [Hyperledger Besu].
 
-This tutorial connects to an ETH2 testnet, and uses [Infura] to access the Goerli ETH1 testnet. You can sign up for a free [Infura] account, or you can start your own [ETH1 Goerli node].
+This tutorial walks you through the process of securely connecting Teku and Web3Signer using TLS, with a focus on creating a keystore and truststore for the Ethereum Holesky testnet. 
 
```

Comment:
```suggestion
```

We can remove this line entirely because the tutorial is already introduced/described in line 11 on this page.

Line 11 actually has some typos, if you could fix to:
> This tutorial configures TLS between Teku and Web3Signer, and uses the [`keytool`](https://docs.oracle.com/en/java/javase/12/tools/keytool.html) utility to generate a keystore and truststore that contains self-signed certificates.